loader/include/Geode/ui/ProgressBarSolid.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <cocos2d.h>
+#include <Geode/ui/ProgressBar.hpp>
+
+namespace geode {
+    // Custom class for the Normal/Practice mode style progress bar
+    class ProgressBarSolid : public ProgressBar {
```

Comment:
Is there a reason for having this be a separate class over just a style parameter on the normal `ProgressBar` class? The only member this adds is the progress bar percentage, which sounds like it would be useful as an option for any progress bar